connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SessionHolder.scala

```diff
@@ -99,12 +101,19 @@ case class SessionHolder(userId: String, sessionId: String, session: SparkSessio
    */
   def classloader: ClassLoader = artifactManager.classloader
 
+  private[connect] def initializeSession(): Unit = {
+    events.postStarted()
+  }
+
   /**
    * Expire this session and trigger state cleanup mechanisms.
    */
   private[connect] def expireSession(): Unit = {
     logDebug(s"Expiring session with userId: $userId and sessionId: $sessionId")
     artifactManager.cleanUpResources()
+    val blockManager = session.sparkContext.env.blockManager
+    blockManager.removeCache(userId, sessionId)
```

Comment:
how is this related to this PR?

Reply:
Looks like this is a bad merge. It in SparkConnectService.RemoveSessionListener at some point, but is no longer there.
Removed